nextflow.config

```diff
@@ -14,7 +14,7 @@ params {
     genome                     = null
     igenomes_base              = null
     igenomes_ignore            = null
-    outdir                     = "./gene_alignment_results"
+    outdir                     = "./${outdir}"
```

Comment:
This change is not needed. This file only defines the default values, but all the parameters can already be set / overridden on the command line with two hyphens, e.g. `--outdir`.
In all my previous tests, I was already able to set the output directory to whatever I wanted with `--outdir`.

Reply:
That makes sense, however, the naming is incorrect anyway as the output is no longer just gene_alignment_data.